.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  release:
+    types: [published]
+
+name: Release
+jobs:
+  pypi:
+    name: Release to PyPI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
why do we need this? should this only get run on master anyways?

Reply:
Doesn't seem to run on master if not included and raises an error in the upload script.

Reply:
[This](https://github.com/FeatureLabs/compose/commit/cb447c428d1ab6c8e571ff30fda04ef669c7fa1a/checks?check_suite_id=237437628) is the raised error when not included.

Reply:
Will this upload the latest commit on master even if the release was a different commit?  Is there a way to get the tag of the release and check out that?

Reply:
No, it will only upload the tagged version of the published release. This section in the logs shows when the upload script checkouts the tagged branch before uploading to PyPI.

```
Release v0.0.2 was published on GitHub ...
HEAD is now at 681298f... update config
...
Uploading distributions to https://test.pypi.org/legacy/
Uploading composeml-0.0.2-py3-none-any.whl
```

Reply:
Got it